.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "airbnb-base",
+  "rules": {
+    "max-len": ["error", {"code": 200, "ignoreComments": true}],
```

Comment:
I have no problem with the line limit here being 200 because of the code generation and trials regarding that. That being said, we should probably keep the 100 line limit on `js-ecqm-engine`'s end, correct?

Reply:
Yes, we should keep it to 100 in `js-ecqm-engine`. I upped it here because the model definitions can get tricky (and, like you said, how the code generation works).